### PR TITLE
[misc] Add memcpy to micro-benchmarks

### PR DESCRIPTION
**Pull request by qiao-bo**
This PR adds a `memcpy` case to the micro-benchmark suites.

### CODE REVIEW

File: benchmarks/microbenchmarks/memcpy.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+from microbenchmarks._items import BenchmarkItem, Container, DataSize, DataType
+from microbenchmarks._metric import MetricType
+from microbenchmarks._plan import BenchmarkPlan
+from microbenchmarks._utils import dtype_size, scaled_repeat_times
```
Comment (yolo2themoon) [inline benchmarks/microbenchmarks/memcpy.py:4]: fill some random data will be better
```suggestion
from microbenchmarks._utils import dtype_size, fill_random, scaled_repeat_times
```

File: benchmarks/microbenchmarks/memcpy.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            dst[I] = src[I]
+
+    repeat = scaled_repeat_times(arch, dsize, repeat)
+    num_elements = dsize // dtype_size(dtype)
+
+    x = container(dtype, num_elements)
+    y = container(dtype, num_elements)
+
+    func = memcpy_field if container == ti.field else memcpy_array
+
```
Comment (yolo2themoon) [inline benchmarks/microbenchmarks/memcpy.py:27]:
```suggestion
    fill_random(x, dtype, container)
```

**Review by yolo2themoon — APPROVED** [review]
LGTM! Thanks!

**Review by lin-hitonami — APPROVED** [review]
LGTM!

**Pull request merged**